config/locales/en.yml

```diff
@@ -1,31 +1,228 @@
-# Files in the config/locales directory are used for internationalization and
```

Comment:
Let's break these long files up like this:
https://guides.rubyonrails.org/i18n.html#organization-of-locale-files

I think that will make it much easier for us to manage them over time, especially when we have multiple languages and we may eventually break up translation tasks (or get GPT to help us :)